src/components/Navbar/styles.module.scss

```diff
@@ -0,0 +1,24 @@
+@import 'scss/variables/_colors';
+
+.navbar-container {
+  border-top: 6px solid $wolox-blue;
+  box-shadow: 0 4px 10px 0 $shadow-box;
+  display: flex;
```

Comment:
remember to use sass global helper classes like this one should be `row center full-width`